package-lock.json

```diff
@@ -1,5 +1,12 @@
 {
   "name": "loopback-next-example",
   "version": "1.0.0",
-  "lockfileVersion": 1
+  "lockfileVersion": 1,
+  "dependencies": {
+    "typescript": {
+      "version": "2.4.1",
+      "resolved": "http://apim-ci1.rtp.raleigh.ibm.com:8080/typescript/-/typescript-2.4.1.tgz",
```

Comment:
Uh oh, this URL does not look right to me. Since this is a public project, it should install dependencies from the public npm repository, not from our internal IBM proxy.